rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -1147,6 +1014,19 @@ public Void apply(NodeSource nodeSource) {
         }, preemptive, isTriggeredFromShutdownHook));
     }
 
+    protected void recoverEligibleNodes(List<RMNode> eligibleNodes) {
```

Comment:
Yes, it looks like but I am afraid that if we call it like a setter, then we will tend to use it afterwards, whereas it should only be used in case of recovery.

Reply:
you could name it setEligibleNodesToRecover ?

Reply:
done